sdk/nodejs/resource.ts

```diff
@@ -269,6 +281,10 @@ export abstract class Resource {
             this.__providers = { ...this.__providers, ...providers };
         }
 
+        // Combine transformations inherited from the parent with transformations provided in opts.
+        const parent = opts.parent || getStackResource() || { __transformations: undefined };
+        this.__transformations = [...(parent.__transformations || []), ...(opts.transformations || [])];
```

Comment:
this is a bit unexpected, but i think i can see why we'd want this.  (we should doc this though).

Is this so you can basically pass transformations to something like Vpc and have them run against the child resources created?

Reply:
Fixed.